game-app/game-headless/src/main/java/org/triplea/game/server/HeadlessLaunchAction.java

```diff
@@ -45,7 +53,7 @@ public void handleGameInterruption(
   @Override
   public void onGameInterrupt() {
     // tell headless server to wait for new connections:
-    HeadlessGameServer.waitForUsersHeadlessInstance();
+    headlessGameServer.waitForUsersHeadless();
```

Comment:
Does this method need to have headless in the name given it's already on a class that has headless in the name?

Reply:
Probably not